main/src/main/java/org/mobilitydata/gtfsvalidator/notice/RouteColorContrastNotice.java

```diff
@@ -0,0 +1,18 @@
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+import org.mobilitydata.gtfsvalidator.type.GtfsColor;
+
+public class RouteColorContrastNotice extends Notice {
```

Comment:
For consistency with the other notices, I suggest reming this one to `RouteColorContrast`.

Another solution would be to append the term "notice" to the other notices' class name:
- `InconsistentAgencyField`
- `StartAndEndDateOutOfOrder`
- `StopTimeWithArrivalBeforePreviousDepartureTime`
- `StopTimeWithDepartureBeforeArrivalTime`
- `StopTimeWithOnlyArrivalOrDepartureTime`
- `UnusedShape`